app/views/docs/email-and-sms-templates.phtml

```diff
@@ -0,0 +1,280 @@
+    </tr>
+    <tr>
+        <td>Subject</td>
+        <td>The title of the email.</td>
+    </tr>
+    <tr>
+        <td>Message</td>
+        <td>The body of the email. You can find the variables available in the <a href="#email-template-syntax">Email Template Syntax</a> section.</td>
+    </tr>
+    </tbody>
+</table>
+<h3><a href="#email-template-syntax" id="email-template-syntax">Email Template Syntax</a></h3>
+<p>Variables can be used in email templates to dynamically format unique emails for each reader. These variables can only be accessed in the <b>Message</b> field of the email template.</p>
+
+<table class="full text-size-small">
+    <thead>
+    <tr>
+        <td style="width: 200px">Variable</td>
+        <td>Description</td>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td><code>{{project}}</code></td>
+        <td>The project name.</td>
+    </tr>
+    <tr>
+        <td><code>{{team}}</code></td>
+        <td>The project team's name.</td>
+    </tr>
+    <tr>
+        <td><code>{{user}}</code></td>
+        <td>The name of the user receiving the email.</td>
+    </tr>
+    <tr>
+        <td><code>{{redirect}}</code></td>
+        <td>The URL for the user to complete the email template's action.</td>
+    </tr>
+    </tbody>
+</table>
+
+<h3><a href="#email-template-examples" id="email-template-components">Email Template Examples</a></h3>
+<p>Here's an example of using these variables in a template.</p>
+<div class="ide margin-bottom" data-lang="html" data-lang-label="html">
+    <pre class="line-numbers"><code class="prism language-html" data-prism>&lt!doctype html&gt
+&lthtml&gt
+
+&lthead&gt
+    &ltstyle&gt
+        ... your style here
+    &lt/style&gt
+&lt/head&gt
+
+&ltbody style="direction: ltr"&gt
+
+&ltdiv style="max-width:650px; word-wrap: break-word; overflow-wrap: break-word;
+  word-break: break-all; margin:0 auto;"&gt
+    &lttable style="margin-top: 32px"&gt
+        &lttr&gt
+            &lttd&gt
+                &lth1&gt
+                    {{subject}}
+                &lt/h1&gt
+            &lt/td&gt
+        &lt/tr&gt
+    &lt/table&gt
+
+    &lttable style="margin-top: 40px"&gt
+        &lttr&gt
+            &lttd&gt
+                &ltp&gtHello &lt/p&gt
+
+                &ltp&gtFollow this link to reset your {{project}} password.&lt/p&gt
+
+                &lta href="{{redirect}}" target="_blank"&gt{{redirect}}&lt/a&gt
+
+                &ltp&gt&ltbr /&gtIf you didn't ask to reset your password, you can ignore this message.&lt/p&gt
+                &ltbr /&gt
+
+                &ltp&gtThanks
+                &ltbr /&gt
+                {{project}} team
+                &lt/p&gt
+            &lt/td&gt
+        &lt/tr&gt
+    &lt/table&gt
+&lt/div&gt
+
+&lt/body&gt
+
+&lt/html&gt</code></pre>
+</div>
+
+<h2><a href="#sms-templates" id="sms-templates">SMS Templates</a></h2>
```

Comment:
We should not mention SMS templates. This section should be gone, and in introduction we shouldn't talk about it.

We disabled this feature for now. We will add it back when we add support for custom SMS provider.